### DISCUSSION TITLE
Can beets search and replace metadata info?

### DISCUSSION THREAD

**Discussion by GUUser91** [beetbox/beets#4568] [Q&A] [answered]
I want to know if it's possible for beets to search for artist info and replace it and if so how do I do it?
For example after importing an album to the database, I want beets to search for the artist called Koji Kondo and replace it with 近藤浩治.
So a song with this artist info will turn from this
`Koji Kondo, Yoshie Arakawa, Yoshinori Kawamoto`
to this
`近藤浩治, Yoshie Arakawa, Yoshinori Kawamoto`

**Comment by arsaboo** [comment, marked as answer]
`beet modify -a albumartist:"Koji Kondo, Yoshie Arakawa, Yoshinori Kawamoto" albumartist="近藤浩治, Yoshie Arakawa, Yoshinori Kawamoto"`
